.github/workflows/industrial_ci.yml

```
@@ -16,10 +16,12 @@ jobs:
   industrial_ci:
     strategy:
       matrix:
-        env:
-          - { ROS_DISTRO: humble, ROS_REPO: ros }
+        ROS_DISTRO: [jazzy]
+        ROS_REPO: [ros]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```

actions/checkoutの最新バージョンが4になっているみたいなので更新してCIが通るか確認をお願いいたします。
https://github.com/marketplace/actions/checkout